pandas/core/frame.py

```diff
@@ -639,15 +639,18 @@ def iterrows(self):
             s = Series(v, index=columns, name=k)
             yield k, s
 
-    def itertuples(self, index=True):
+    def itertuples(self, index=True, name="Pandas"):
         """
-        Iterate over the rows of DataFrame as tuples, with index value
-        as first element of the tuple.
+        Iterate over the rows of DataFrame as namedtuples, with index value
+        as first element of the tuple. With a large number of columns (>255),
```

Comment:
add a ``Note`` section where you mention the renaming of pos args and the large number of columns issue (this goes after the Parameters section)